pwiz_tools/Skyline/SettingsUI/PeptideSettingsUI.cs

```diff
@@ -1258,8 +1296,8 @@ public IFormView ShowingFormView
 
         public TABS SelectedTab
         {
-            get { return (TABS)tabControl1.SelectedIndex; }
-            set { tabControl1.SelectedIndex = (int)value; }
+            get { return ControlIndexToTab(tabControl1.SelectedIndex); }

```

Comment:
I think you could just make this:
return tabMap[tabControl1.SelectedTab];

Reply:
The enum is used in serialization, so we still need a mapping, but I agree that I could simplify this code by just using a dictionray (and matching reverse dictionary). Thanks as always for a thoughtful review.